pydov/types/interpretaties.py

```diff
@@ -204,3 +204,198 @@ def _parse_xml_data(self):
+        super(HydrogeologischeStratigrafieLaag, self).__init__(
+            'hydrogeologische_interpretatie_laag')
+
+    @classmethod
+    def from_xml_element(cls, element):
+        """Build an instance of this subtype from a single XML element.
+
+        Parameters
+        ----------
+        element : etree.Element
+            XML element representing a single record of this subtype.
+
+        """
+        laag = HydrogeologischeStratigrafieLaag()
+
+        for field in cls.get_fields().values():
+            laag.data[field['name']] = laag._parse(
+                func=element.findtext,
+                xpath=field['sourcefield'],
+                namespace=None,
+                returntype=field.get('type', None)
+            )
+
+        return laag
+
+
+class HydrogeologischeStratigrafie(AbstractDovType):
+    """Class representing the DOV data type for boreholes."""
+
+    _subtypes = [HydrogeologischeStratigrafieLaag]
+
+    _fields = [{
+        'name': 'pkey_interpretatie',
+        'source': 'wfs',
+        'sourcefield': 'Interpretatiefiche',
+        'type': 'string'
+    }, {
+        'name': 'pkey_boring',
+        'source': 'custom',
+        'type': 'string',
+        'definition': 'URL die verwijst naar de gegevens van de boring '
+                      'waaraan deze hydrogeologische stratigrafie gekoppeld is '
+                      '(indien gekoppeld aan een boring).',
+        'notnull': False
+    }, {
+        'name': 'pkey_sondering',
+        'source': 'custom',
+        'type': 'string',
+        'definition': 'URL die verwijst naar de gegevens van de sondering '
+                      'waaraan deze informele stratigrafie gekoppeld is ('
+                      'indien gekoppeld aan een sondering).',
+        'notnull': False
+    }, {
+        'name': 'betrouwbaarheid_interpretatie',
+        'source': 'wfs',
+        'sourcefield': 'Betrouwbaarheid',
+        'type': 'string'
+    },  {
+        'name': 'x',
+        'source': 'wfs',
+        'sourcefield': 'X_mL72',
+        'type': 'float'
+    }, {
+        'name': 'y',
+        'source': 'wfs',
+        'sourcefield': 'Y_mL72',
+        'type': 'float'
+    }]
+
+    def __init__(self, pkey):
+        """Initialisation.
+
+        Parameters
+        ----------
+        pkey : str
+            Permanent key of the Boring (borehole), being a URI of the form
+            `https://www.dov.vlaanderen.be/data/boring/<id>`.
+
+        """
+        super(HydrogeologischeStratigrafie, self).__init__(
+            'interpretatie', pkey)
+
+    @classmethod
+    def from_wfs_element(cls, feature, namespace):
+        """Build `Boring` instance from a WFS feature element.
+
+        Parameters
+        ----------
+        feature : etree.Element
+            XML element representing a single record of the WFS layer.
+        namespace : str
+            Namespace associated with this WFS featuretype.
+
+        Returns
+        -------
+        boring : Boring
```

Comment:
Should be HydrogeologischeStratigrafie

Reply:
Ok, thx for the review, changed all docstrings in new commit